### PR TITLE
[ETFE-3093] - Align confirmation page

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

File: app/controllers/ConfirmationController.scala [hunk 1]
```diff
@@ -55,4 +58,12 @@ class ConfirmationController @Inject()(
...
       }
     }
+
+  //TODO: could be made simpler when consignee flow built?
+  private def hasConsigneeChanged()(implicit request: DataRequest[_]): Boolean =
+    request.userAnswers.get(ConsigneeExcisePage) != ConsigneeExcisePage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeExemptOrganisationPage) != ConsigneeExemptOrganisationPage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeBusinessNamePage) != ConsigneeBusinessNamePage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeAddressPage) != ConsigneeAddressPage.getValueFromIE801 ||
+      request.userAnswers.get(ConsigneeExportVatPage) != ConsigneeExportVatPage.getValueFromIE801
```
Comment (MJCallahanPage) [inline app/controllers/ConfirmationController.scala:68]: Move to `ConsigneeSection`

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
One small thing as discussed.

File: app/controllers/ConfirmationController.scala (hunk 1 above)
Comment (LewisMitchell) [reply app/controllers/ConfirmationController.scala]: Done in [19d004f](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/23/commits/19d004f1f89c3d1be1adc6408ecf135d81e5e31d)

**Review by MJCallahanPage — APPROVED** [review]
LGTM

**Pull request merged**